### PR TITLE
Add Buy ETH button to bottom of Wallet tab #5398

### LINKED ISSUE #5398: Add Buy ETH button to bottom of Wallet tab
* Only if Ethereum mainnet is enabled
* Works like the "Buy" button when we tap "Buy" after tapping the ETH token > … > Buy

Something like Android's:

### PR DESCRIPTION
**Pull request by oa-s**
Closes #5398

### CODE REVIEW

File: AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift [hunk 1]
```diff
@@ -188,6 +188,12 @@ extension UIPasteboard {
 
 extension TokensCoordinator: TokensViewControllerDelegate {
 
+    func buyCryptoSelected(in viewController: UIViewController) {
+        guard config.enabledServers.contains(.main) else { return }
```
Comment (hboon) [inline AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift:192]: Shall we move this condition to control whether the buy button is shown?

**Review by hboon — COMMENTED** [review]
It clips off the bottom of the table — in the screenshot, there is a GNO token below the xDai token:

Screenshot 2022-10-05 at 11 13 09 AM

File: AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift (hunk 1 above)
Comment (oa-s) [reply AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift:192]: checking

**Comment by oa-s** [comment]
@hboon  updated pr, looks ok now

**Comment by hboon** [comment]
Just noticed 1 thing, the background colors of the actionsheet buttons seems a bit strange. But maybe it was always there with the Buy Crypto buttons and not because of this PR?

Screenshot 2022-10-05 at 7 36 08 PM

**Comment by oa-s** [comment]
not sure that its pr changes, guess it was before

**Comment by hboon** [comment]
Yeah. Thought might be it. Let’s  fix it separately.

**Pull request merged**